Proxy/HTTP/FilterAuthenticatedNonBearerTokens.bambda

```diff
@@ -0,0 +1,56 @@
+/**
+ * Filter when an Authorization header is present, not empty and does not include a traditional bearer token (beginning with "ey")
+ *
+ * @author GangGreenTemperTatum (https://github.com/GangGreenTemperTatum)
+ **/
+
+var configNoFilter = true;
+var configNotInScopeOnly = true;
```

Comment:
We'd recommend renaming this variable to `configInScopeOnly` and setting to `false` to make this Bambda easier to read.